doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -557,6 +557,10 @@ Libraries for networking
 
 * :ref:`lib_aws_fota` library:
 
+  * Added:
+
+    * Support for a single ``url`` field in job documents.
```

Comment:
I feel like this needs a little more explanation. Something like "instead of an array" or something along those lines.

Reply:
How about the following:

`Support for a single ``url`` field in job documents. Previously the hostname and path of the download URL could only be specified separately. `

Reply:
That sounds good. Remember to have both sentences on their own lines.